audio-worklet/basic/message-port.html

```diff
@@ -50,6 +56,9 @@ <h1>
         this.counter++;
         console.log('[Node:Received] "' + event.data.message +
                     '" (' + event.data.timeStamp + ')');
+        
+        // Notify the processor when the node gets 10 messages. Then reset the
+        // counter.
```

Comment:
How does this match up with the description in message-port.html?

Reply:
I added one more item in the description.